llvm/include/llvm/Transforms/Instrumentation/RealtimeSanitizer.h

```diff
@@ -0,0 +1,35 @@
+//===--------- Definition of the RealtimeSanitizer class ---------*- C++
```

Comment:
Some `*Sanitizer.h` headers had non-conventional headers, which might lead to confusion. I fixed them in 
c6a3f4e2f2586cb1ce51306c305752a78bdba263